class/SyncHandlers/SyncHandlerBase.php

```diff
@@ -244,23 +286,10 @@ protected static function sync_all_paginated(string $url, int $page_number = 1,
     return;
   }
 
-  protected static function check_sync_of_authors($postarr)
+  public static function prevent_content_sanitization($postarr)
   {
-    if ($postarr["post_type"] === PASSLESYNC_POST_TYPE) {
-      $meta = $postarr["meta_input"];
-      $all_author_shortcodes_on_post = array_merge($meta["post_author_shortcodes"], $meta["post_coauthors_shortcodes"]);
-
-      $wp_author_posts = get_posts([
-        'post_type' => PASSLESYNC_AUTHOR_TYPE,
-      ]);
-
-      foreach ($all_author_shortcodes_on_post as $author_shortcode) {
-        $wp_post_by_author = wp_filter_object_list($wp_author_posts, array('post_name' => $all_author_shortcodes_on_post));
-
-        if (!$wp_post_by_author) {
-          PeopleWebhookActions::update($author_shortcode);
-        }
-      }
-    }
+    // Remove the default content sanitization to allow hidden field tags
+    remove_filter('content_save_pre', 'wp_filter_post_kses');
+    return $postarr;
   }
```

Comment:
I am not sure this needs to be a method that accepts $postarr as an argument as it doesn't seem to be using $post arr at all.
Also, the filter that it removes, should probably be re-added after the post is inserted so this line should probably appear before insert post, then insert post should happen and then this filter should be re-instated.